lxd/images.go

```diff
@@ -106,7 +106,7 @@ func unpack(file string, path string) error {
 	if err != nil {
 		shared.Debugf("Unpacking failed")
 		shared.Debugf(string(output))
-		return err
+		return fmt.Errorf("Unpack failed, %s.", err)
```

Comment:
On Wed, Jul 20, 2016 at 11:19:19AM -0700, Sean Christopherson wrote:
> > @@ -106,7 +106,7 @@ func unpack(file string, path string) error {
> >  	if err != nil {
> >  		shared.Debugf("Unpacking failed")
> >  		shared.Debugf(string(output))
> > -		return err
> > +		return fmt.Errorf("Unpack failed, %s.", err)
> 
> > Do we have verbose on where we shouldn't?
> 
> I don't think so.  On my system tar complains about every single failure once the disk runs out of space, i.e. output has hundreds of lines complaining about "not enough space".  I'm hesitant to capture only the tail of the output in case the original failure was somehow different than subsequent failures.

Ah, I see. The first few lines sounds good to me then too!

Tycho

> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/2222/files/16015f0ab699178f206e4cc9e2c4a4b6e182d976#r71578636